SofaGLFW/src/SofaGLFW/SofaGLFWBaseGUI.h

```diff
@@ -138,11 +143,19 @@ class SOFAGLFW_API SofaGLFWBaseGUI
     int m_lastWindowPositionY{ 0 };
     int m_lastWindowWidth{ 0 };
     int m_lastWindowHeight{ 0 };
+    double lastProjectionMatrix[16];
+    double lastModelviewMatrix[16];
```

Comment:
```suggestion
    double lastProjectionMatrix[16];
    double lastModelviewMatrix[16];
```
Prefer to use `sofa::type::fixed_array`